### PR TITLE
Non-ASCII characters problem in Institute Leaderboard solved

### PR DESCRIPTION
**Pull request by chotuaashiq**
Solves issue #264 

Modified the query to the database for fetching the result of the institute leaderboard.

### CODE REVIEW

File: controllers/default.py [hunk 1]
```diff
@@ -615,7 +616,7 @@ def leaderboard():
 
     if request.vars.has_key("q") and request.vars["q"]:
         heading = T("Institute Leaderboard")
-        institute = request.vars["q"]
+        institute = urllib.unquote(request.vars["q"])
```
Comment (raj454raj) [inline controllers/default.py]: please import urllib just before this statement instead of across the controller

File: controllers/default.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 import parsedatetime as pdt
 import requests
 import utilities
+import urllib
```
Comment (raj454raj) [inline controllers/default.py]: remove the extra space from the end of the line

File: controllers/default.py (hunk 1 above)
Comment (chotuaashiq) [reply controllers/default.py]: @raj454raj changed the import location of the library

**Comment by raj454raj** [comment]
@chotuaashiq : This is perfect! But we also need to handle it in js.
 Handle it in a generalized way [here](https://github.com/stopstalk/stopstalk-deployment/blob/master/static/js/appjs/default/leaderboard.js#L19).
You can see the issue if you toggle the leaderboard switch between Global and Friends after applying the institute filter.

**Comment by raj454raj** [comment]
@chotuaashiq : Still working on this ?

**Comment by raj454raj** [comment]
Fixed in https://github.com/stopstalk/stopstalk-deployment/commit/9ad5120cf91af3ff9aa004023b4107d980a08c5b

**Pull request merged**